test/server/cards/02_SHD/units/SalaciousCrumbObnoxiousPet.spec.ts

```diff
@@ -62,6 +62,9 @@ describe('Salacious Crumb, Obnoxious Pet', function() {
                 const { context } = contextRef;
 
                 context.salaciousCrumb.exhausted = true;
+                context.player1.passAction();
+                context.player2.clickCard(context.frontierAtrt);
+                context.player2.clickCard(context.p1Base);
```

Comment:
What's this for?

Reply:
Okay so this is an interesting case. Since Player1 is in the action phase SalaciousCrumb is added as a selectableCard before the exhausted = true. Because he is added as a selectable card the prompt before and after expect(context.salaciousCrumb).not.toHaveAvailableActionWhenClickedBy(context.player1);
is different which fails the check. I'm open to any ideas btw.